doc/build.md

```diff
@@ -2,121 +2,193 @@
+
+```
+➜  ctest -j 8
+Test project /Users/sabrih/Desktop/workspace/build/Debug/plugin/al
+    Start 4: AL_USDMayaTestPlugin
+    Start 5: TestUSDMayaPython
+    Start 8: TestPxrUsdTranslators
+    Start 7: TestAdditionalTranslators
+    Start 1: AL_MayaUtilsTests
+    Start 3: Python:AL_USDTransactionTests
+    Start 2: GTest:AL_USDTransactionTests
+    Start 6: testMayaSchemas
+1/8 Test #2: GTest:AL_USDTransactionTests .....   Passed    0.06 sec
+2/8 Test #6: testMayaSchemas ..................   Passed    0.10 sec
+3/8 Test #3: Python:AL_USDTransactionTests ....   Passed    0.73 sec
+4/8 Test #1: AL_MayaUtilsTests ................   Passed    6.01 sec
+5/8 Test #8: TestPxrUsdTranslators ............   Passed    9.96 sec
+6/8 Test #5: TestUSDMayaPython ................   Passed   10.28 sec
+7/8 Test #7: TestAdditionalTranslators ........   Passed   12.06 sec
+8/8 Test #4: AL_USDMayaTestPlugin .............   Passed   27.43 sec
+100% tests passed, 0 tests failed out of 8
+```
+
+# Building Pixar USD
 
-When building Pixar USD, you need to pass ```--no-maya``` flag to ensure that ```third_party/maya``` plugin doesn't get built since this plugin is now part of maya-usd.
+##### Flags, Version
 
-It is important that the version between ```Pixar USD``` and ```Maya USD plugin```match. We are currently using ```v19.05``` for Maya plugin in ```master``` branch and therefore ```Pixar USD``` build should match this version.
+Pixar has recently removed support for building Maya USD libraries and plug-ins in their ```build_usd.py```. When building USD, it is important that the ```Pixar USD``` and ```Maya USD plug-in``` versions match. Consult the source versions table in the [Developer](DEVELOPER.md) document. 
 
 ##### Boost:
 
-Currently Animal Logic plugin has a dependency on some of the boost components ( e.g thread, filesystem ). When building Pixar USD, one needs to pass following key,value paired arguments for boost to include those components: 
+Currently the Animal Logic plugin has a dependency on some of the boost components ( e.g thread, filesystem ). When building Pixar USD, one needs to pass the following key,value paired arguments for boost to include those components: 
 e.g 
 
-```python build_usd.py --build-args boost,"--with-date_time --with-thread --with-system --with-filesystem" --no-maya ~/Desktop/BUILD```
+```
+python build_usd.py ~/Desktop/BUILD --build-args boost,"--with-date_time --with-thread --with-system --with-filesystem" 
+```
 
 ##### Python:
 
 Some DCCs (most notably, Maya) may ship with and run using their own version of Python. In that case, it is important that USD and the plugins for that DCC are built using the DCC's version of Python and not the system version. Note that this is primarily an issue on macOS, where a DCC's version of Python is likely to conflict with the version provided by the system. 
```

Comment:
The way this was worded made a more sense when it was in the core USD repo. The DCC in this context is obviously Maya, so this could probably be adjusted. Just dropping the first sentence and replacing "DCC" with "Maya" will probably get most of the way there.